database/index.js

```diff
@@ -0,0 +1,44 @@
+//in here set up mongoose database
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+// var bodyParse = require('body-parser');
```

Comment:
I believe it's good practice to get rid of commented-out "zombie" code

Reply:
Cleaned up